core/src/main/java/org/geysermc/geyser/translator/collision/GlassPaneCollision.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2019-2022 GeyserMC. http://geysermc.org
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @author GeyserMC
+ * @link https://github.com/GeyserMC/Geyser
+ */
+
+package org.geysermc.geyser.translator.collision;
+
+import lombok.EqualsAndHashCode;
+import org.geysermc.geyser.level.physics.BoundingBox;
+import org.geysermc.geyser.session.GeyserSession;
+
+@EqualsAndHashCode(callSuper = true)
+@CollisionRemapper(regex = "glass_pane$", usesParams = true, passDefaultBoxes = true)
+public class GlassPaneCollision extends BlockCollision {
+    /**
+     * 1 = north
+     * 2 = east
+     * 3 = south
+     * 4 = west
+     * b1 = north, east
+     * b2 = east, south
+     * b3 = south, west
+     * b4 = west, north
+     */
+    private int facing;
+    private int facingB;
```

Comment:
Is there a need for a second int?
I'd imagine we could re-use the `facing` variable (with e.g. values 5,6,7,8 for the corners), since there shouldn't be a case where both variables are used at once (making the other int always 0).